dll/ntdll/rtl/version.c

```diff
@@ -197,8 +146,14 @@ RtlGetVersion(IN OUT PRTL_OSVERSIONINFOW lpVersionInformation)
         InfoEx->wProductType = SharedUserData->NtProductType;
         InfoEx->wReserved = 0;
 
-        /* HACK: ReactOS specific changes, see bug-reports CORE-6611 and CORE-4620 (aka. #5003) */
-        SetRosSpecificInfo(InfoEx);
+        if (InfoEx->wProductType == VER_NT_SERVER)
```

Comment:
We might want to keep a comment that this is reactos specific?

Reply:
OK.